src/wizard/javascript/performance-onboarding/javascript/1.install.md

```diff
@@ -0,0 +1,18 @@
+---
+name: JavaScript
+doc_link: https://docs.sentry.io/platforms/javascript/performance/
+support_level: production
+type: language
+---
+
+#### Install
```

Comment:
Is there a specific reason we're using heading level 4 here? Is there a reason it isn't level 2? This applies to all of these files.

Reply:
@imatwawana This was intentional so that we have the "right" header sizing that fits well with the inserted checkboxes: 

<img width="450" alt="Screen Shot 2022-05-13 at 2 53 51 PM" src="https://user-images.githubusercontent.com/139499/168354231-94555cad-8f73-4322-af4e-926a98293381.png">